pubtools/_impl/pluggy.py

```diff
@@ -24,7 +24,8 @@ def resolve_hooks():
     #
     for ep in pkg_resources.iter_entry_points("console_scripts"):
         if ep.module_name.startswith("pubtools"):
-            ep.resolve()
+            # Method resolve() was introduced in setuptools 11.3 to replace load()
+            getattr(ep, "resolve", lambda: ep.load(require=False))()
```

Comment:
Minor: probably more readable to split the "get load function" and "call it" steps to separate statements like:

```python
resolve = getattr(ep, "resolve", ...)
resolve()
```